gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/language/grammar/DefaultGremlinBaseVisitor.java

```diff
@@ -987,16 +1003,31 @@ protected void notImplemented(final ParseTree ctx) {
 	 * {@inheritDoc}
 	 */
 	@Override
```

Comment:
This file contains inconsistencies on the placement of the Override annotation. In some places its inline with the method signature, in others, its on a separate line. You should keep these in the same line to match the rest of the file.

Reply:
Prob copy and past error, I'll fix them